spacy_llm/tasks/__init__.py

```diff
@@ -8,6 +9,7 @@
 from .textcat import TextCatTask, make_textcat_task
 
```

Comment:
We should merge in `main` and add the entity linker to `_LATEST_TASKS`.
(don't merge in `main` directly - let's sync `develop` first with `main`, then merge in `develop` in this PR.

Reply:
Done in https://github.com/explosion/spacy-llm/pull/241/commits/88ad945fa434a60423aef8030b8597c618dfb516.